homeassistant/components/here_travel_time/config_flow.py

```diff
@@ -150,37 +164,49 @@ async def async_step_origin_coordinates(
             self._config[CONF_ORIGIN_LONGITUDE] = user_input[CONF_ORIGIN][
                 CONF_LONGITUDE
             ]
+            self._config.pop(CONF_ORIGIN_ENTITY_ID, None)
             return await self.async_step_destination_menu()
         schema = vol.Schema(
             {
                 vol.Required(
                     CONF_ORIGIN,
                     default={
```

Comment:
Instead of defaults, use constant schemas with suggested values added by calling `self.add_suggested_values_to_schema`